package.json

```diff
@@ -22,7 +22,7 @@
   "devDependencies": {
     "babel-eslint": "^6.1.0",
     "snazzy": "^4.0.0",
-    "standard": "^7.1.2"
+    "standard": "^8.0.0"
```

Comment:
Unless there's a good reason for bumping this in this PR, I'd rather leave this as is for now.